torch/fx/experimental/symbolic_shapes.py

```diff
@@ -148,8 +149,13 @@ class SymNode:
     This is a type erased SymInt/SymFloat which we use to do actual operations.
     End users don't touch this.  Magic methods are NOT defined on this object.
     """
-    def __init__(self, expr, shape_env, pytype, constant=None):
+    def __init__(self, expr, symbol, shape_env, pytype, constant=None):
         self._expr = expr
+        # Unlike expr, sympy.Symbol is guaranteed to be a symbol,
```

Comment:
For intermediate values, what is it then? None?

Seems like it - probably a bad annotation then.

Reply:
you're right